packages/next/client/page-loader.js

```diff
@@ -98,7 +98,7 @@ export default class PageLoader {
         if (process.env.__NEXT_GRANULAR_CHUNKS) {
           this.getDependencies(route).then(deps => {
             deps.forEach(d => {
-              if (!document.querySelector(`script[src^="${d}"]`)) {
+              if (!/\.css$/.test(d) && !document.querySelector(`script[src^="${d}"]`)) {
```

Comment:
Would it be better to test that it ends with `.js`? Users can customize their webpack config to emit all sorts of things...

Reply:
Right now the build-manifest-plugin filters out everything except css and JS from the client side manifest, but sure--I'm fine making that change for future-proofing.